static/app/components/eventOrGroupExtraDetails.tsx

```diff
@@ -74,66 +78,86 @@ function EventOrGroupExtraDetails({
     organization.features.includes('session-replay') &&
     projectCanLinkToReplay(organization, project);
 
-  return (
-    <GroupExtra>
-      {shortId && (
-        <InboxShortId
-          shortId={shortId}
-          avatar={
-            project && (
-              <ShadowlessProjectBadge project={project} avatarSize={12} hideName />
-            )
-          }
+  const hasNewLayout = organization.features.includes('issue-stream-table-layout');
+  const {subtitle} = getTitle(data);
+
+  const level = 'level' in data ? data.level : null;
+
+  const items = [
+    shortId ? (
+      <InboxShortId
+        shortId={shortId}
+        avatar={
+          project && <ShadowlessProjectBadge project={project} avatarSize={12} hideName />
+        }
+      />
+    ) : null,
+    hasNewLayout && level ? <ErrorLevelText level={level} /> : null,
```

Comment:
Nit: the color of this text semes to be solid black, but the designs appear to be `subtext` colored. 

Current: 
<img width="67" alt="image" src="https://github.com/user-attachments/assets/675adb06-9d93-40ac-8e96-ed3b513e641b">

Designs:
![image](https://github.com/user-attachments/assets/f8e6e5d5-4a38-4841-a7b7-d09425627e69)

Reply:
Good catch! Yeah there are a few things in that row that aren't colored correctly, but the PR was getting pretty large so I decided to leave that work for later